be/src/exec/parquet_writer.h

```diff
@@ -42,6 +42,7 @@ struct TableInfo {
     bool _enable_dictionary = true;
     std::string _partition_location = "";
     std::shared_ptr<::parquet::schema::GroupNode> _schema;
+    TCloudConfiguration _cloud_conf;
```

Comment:
use `TCloudConfiguration&` instead?

Reply:
this could complicate the lifetime relation since tableInfo does not own cloud_conf